### ISSUE TITLE
Default resources are too small

### ISSUE THREAD

**Issue by spikecurtis** [coder/enterprise-helm#275]
We seem to have

```
resources:
  requests:
    cpu: "250m"
    memory: "512Mi
  limits:
    cpu:  "250m"
    memory: "512Mi"
```

Should start with

```
resources:
  requests:
    cpu: "1000m"
    memory: "1Gi
  limits:
    cpu:  "1000m"
    memory: "1Gi"
```

Even 50 users can max out a core if the use web proxying extensively.

**Comment by ericpaulsen** [comment]
related: https://github.com/coder/enterprise-helm/pull/254

i will get this PR merged.

**Comment by ericpaulsen** [comment]
closed via #254
